lib/internal/streams/utils.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
 const {
-  Symbol,
   SymbolAsyncIterator,
+  SymbolFor,
   SymbolIterator,
 } = primordials;
 
-const kDestroyed = Symbol('kDestroyed');
-const kIsErrored = Symbol('kIsErrored');
-const kIsReadable = Symbol('kIsReadable');
-const kIsDisturbed = Symbol('kIsDisturbed');
+const kIsDestroyed = SymbolFor('node.stream.__PRIVATE__kIsDestroyed');
+const kIsErrored = SymbolFor('node.stream.__PRIVATE__kErrored');
+const kIsReadable = SymbolFor('node.stream.__PRIVATE__kReadable');
+const kIsWritable = SymbolFor('node.stream.__PRIVATE__kWritable');
+const kIsDisturbed = SymbolFor('node.stream.__PRIVATE__kDisturbed');
```

Comment:
Sorry I think about this only now, but we should probably have a comment that explains why we're using `SymbolFor` and leave those symbols undocumented yet "public".